### PR TITLE
Implement Google Tag Manager

### PR DESCRIPTION
**Pull request by ollietreend**
This branch implements Google Tag Manager and removes manual Google Analytics tracking.

### Other Changes

The accordion component has been refactored to use HTML5's `<details>` element. This required changes to the associated SASS and JavaScript.

Additionally, the bug which caused Google Analytics events to include Google Translated accordion titles has been rectified. Analytics events will now always track the original accordion title, even if the text has been translated in the user's browser.

### CODE REVIEW

File: web/app/themes/ppj/src/vue/JobSummary.vue [hunk 1]
```diff
@@ -47,9 +48,12 @@
...
-      handleViewJobClick: function() {
-        ga('send', 'event', 'ViewJob', 'ViewJob',this.prisonName ,5);
-        window.location = this.url;
+      pushEventToGtm: function() {
+        const gtmEvent = {
+          event: 'view_job_click',
+          prisonName: this.prisonName
+        };
+        window.dataLayer.push(gtmEvent);
       }
```
Comment (jsmcgd) [inline web/app/themes/ppj/src/vue/JobSummary.vue]: Minor thought - would it be preferable to have a non-generic name for pushEventToGtm given that the functionality of the method is non-generic?

Comment (ollietreend) [reply web/app/themes/ppj/src/vue/JobSummary.vue]: I did consider that – and then thought that I might be overthinking it.

Any suggestions on a better name for that method? I was thinking along the lines of `gtmViewJobClickEvent` or something.

Comment (jsmcgd) [reply web/app/themes/ppj/src/vue/JobSummary.vue]: I was thinking either to have a handleClickEvent wrapper or perhaps call it something like pushJobClickEventToGtm?

Comment (jsmcgd) [reply web/app/themes/ppj/src/vue/JobSummary.vue]: Or what you suggested :)

**Pull request merged**